test/step-function.test.ts

```diff
@@ -1519,6 +1519,37 @@ test("return AWS.SDK.CloudWatch.describeAlarms dynamic parameters", () => {
   expect(normalizeDefinition(definition)).toMatchSnapshot();
 });
 
+test("return AWS.SDK.ApiGatewayManagementApi.postToConnection is not supported", () => {
+  const { stack } = initStepFunctionApp();
+  expect(
+    () =>
+      new ExpressStepFunction<
+        { prefix: string | undefined },
+        AWS.CloudWatch.MetricAlarms | undefined
+      >(stack, "fn", async (input) => {
+        await $AWS.SDK.ApiGatewayManagementApi.postToConnection(
+          {
+            ConnectionId: "blah",
+            Data: "some data",
+          },
+          {
+            iam: {
+              resources: ["*"],
+            },
+          }
+        );
```

Comment:
I thought we were going to polyfill this? Or are we planning to implement a different API call?

Reply:
I was starting to look at implementing `API.Invoke`. We can add a convenience method for connections along with it, maybe.

`API.postConnection`
`API.getConnection`
`API.deleteConnection`

Reply:
To track.
https://github.com/functionless/functionless/issues/457